backend/app/services/message_template_service.py

```diff
@@ -57,22 +60,23 @@ async def get_all_templates(self) -> List[MessageTemplate]:
                 id=row['id'],
                 template_name=row['template_name'],
                 template_text=row['template_text'],
+                admin_user_id=row['admin_user_id'],
                 created_at=row['created_at'],
                 updated_at=row['updated_at'],
                 variables=variables
             ))
 
         return templates
 
-    async def get_template(self, template_id: str) -> Optional[MessageTemplate]:
+    async def get_template(self, template_id: str, auth: AuthenticatedUser) -> Optional[MessageTemplate]:
         """Get a specific message template"""
         query = """
-            SELECT id, template_name, template_text, created_at, updated_at
+            SELECT id, template_name, template_text, admin_user_id, created_at, updated_at
             FROM message_templates
-            WHERE id = ?
+            WHERE id = ? AND admin_user_id = ?
```

Comment:
same here 
i need to get all templates
not just the ones I created
my templates should be accessible to all community admins